add-on/preferences/prefs.js

```diff
@@ -34,3 +33,6 @@ pref("loop.facebook.enabled", true);
 #else
 pref("loop.facebook.enabled", false);
 #endif
+pref("loop.facebook.appId", "1519239075036718");
+pref("loop.facebook.shareUrl", "https://www.facebook.com/dialog/send?app_id=%APP_ID%&link=%ROOM_URL%&redirect_uri=%REDIRECT_URI%");
```

Comment:
I'm ok with having a separate `appId` but can understand that it's unnecessary so let's get rid off it as Ed has suggested